README.md

```diff
@@ -73,9 +71,15 @@ as `error` level logs.
 
 ##### Methods
 
-#### `logger.report(level, report[, state])`
+#### `logger.info(action[, state])`
```

Comment:
I think you should use six `#` - `######`. Not sure what the formatting looks like

Reply:
Are these methods generic? Like can they be used without redux as well?

Reply:
yes, totally

Reply:
Isn't this going back to having params that are redux specific?

Reply:
sorry, forgot to commit README file 🙈